### PR TITLE
Remove global usage in GisDataEditorController

### PR DESCRIPTION
**Pull request by MoonE**

### CODE REVIEW

File: libraries/classes/Controllers/GisDataEditorController.php [hunk 1]
```diff
@@ -56,12 +44,9 @@ public function __invoke(ServerRequest $request): void
...
 
         // Get data if any posted
-        $GLOBALS['gis_data'] = [];
-        if (is_array($gisDataParam)) {
-            $GLOBALS['gis_data'] = $gisDataParam;
-        }
+        $gisData = is_array($gisDataParam) ? $gisDataParam : [];
 
-        $GLOBALS['gis_types'] = [
+        $gisTypes = [
```
Comment (kamil-tekiela) [inline libraries/classes/Controllers/GisDataEditorController.php:49]: Is this a constant?

Comment (MoonE) [reply libraries/classes/Controllers/GisDataEditorController.php:49]: Yeah, seems to be one.

Comment (kamil-tekiela) [reply libraries/classes/Controllers/GisDataEditorController.php:49]: Maybe move it to a proper constant?

Comment (MoonE) [reply libraries/classes/Controllers/GisDataEditorController.php]: Done

**Review by MauricioFauth — APPROVED** [review]
:100:

**Pull request merged**